tests/e2e/specs/menu-top/MenuTop.spec.js

```diff
@@ -67,5 +67,27 @@ describe('components/menu-top/MenuTop', () => {
       cy.get('[data-test="grapher--grid-horizontal"]')
         .should('not.exist');
     });
+
+    it('can toggle yardlines', () => {
```

Comment:
In this test, can we also verify that the number of vertical grid lines increased?

Reply:
^ bump
We test this in the unit tests but we should include this in the e2e tests as well